<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaxMinByBrandDialog</class>
 <widget class="QDialog" name="MaxMinByBrandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>155</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Визначення найдешевшого та найдорожчого велосипеда</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="editBrand"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonFindCheapAndExpensive">
         <property name="text">
          <string>Знайти найдешевший та найдорожчий</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Найдешевша модель:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editCheapest">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Найдорожчка модель:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editMostExpensive">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
